components/TheImageUploadForm.vue

```diff
@@ -171,29 +186,48 @@ export default {
     },
     async submit() {
       if (this.$refs.form.validate()) {
+        this.isError = false;
         this.isUploading = true;
+
         const assetInfo = {
           assetFile: this.file,
           description: this.description.trim(),
           license: this.license,
           tags: this.tags,
           wallet: ethUtil.getWallet(),
         };
-        const payload = await assetUtil.formatAndSignAsset(
-          assetInfo,
-          'Upload asset with following information'
-        );
-
-        const params = new FormData();
-        Object.keys(payload).forEach((key) => {
-          params.append(key, payload[key]);
-        });
-        axios
-          .post('/api/assets/upload', params)
-          .then((mediaObj) => mediaObj) // TODO: go to my image page maybe?
-          .catch((err) => {
-            console.error(err);
+
+        try {
+          let payload;
+          try {
```

Comment:
I think no need to nest try catch block.

Reply:
You prefer if block?

Reply:
@inDream You prefer if block?